docker-compose.yml

```
@@ -0,0 +1,41 @@
+services:
+  app:
+    container_name: app 
```

yamllint

[error] trailing spaces

(trailing-spaces)